system/shaders/yuv2rgb_basic.glsl

```diff
@@ -79,12 +79,11 @@ vec4 process()
   rgb.a = gl_Color.a;
 #endif
 
-#elif defined(XBMC_NV12_RRG)
+#elif defined(XBMC_NV12)
```

Comment:
I doubt that this path works for sw decoding. gles uses still LUMINANCE and ALPHA_LUMINANCE textures